variable/replace.go

```diff
@@ -21,7 +25,8 @@ func (r *variableInstance) ReplaceVariables(data string) (string, error) {
 		replacement, err := r.ParseVariable(removeBraces(v))
 		if err != nil {
 			log.Printf("Error parsing variable %s: %s", v, err)
```

Comment:
Rather than logging (packages like this shouldn't log!), you can return a special error type. That way, users can check if the error type is a certain type and then take appropriate action. Logging can be done at the `app/` level where the logger has already been initiated.

Reply:
Thanks for pointing this out. I still wanted the function to continue on error so I think I've found a solution to combine all the errors before exiting.